### DISCUSSION TITLE
how to change the time zone?

### DISCUSSION THREAD

**Discussion by efrain2007** [SeaQL/sea-orm#525] [Q&A]
![image](https://user-images.githubusercontent.com/65697999/152605687-51879d65-0e0a-4dbd-8592-5bc533c91b01.png)
I want to change the time zone to connect in another zone

**Comment by billy1624** [comment]
Hey @efrain2007, you can check this comment made by SQLx team.
- https://github.com/launchbadge/sqlx/issues/329#issuecomment-631313940

**Comment by gudaoxuri** [comment]
You can handle it yourself before sqlx provides official support：

https://github.com/ideal-world/tardis/blob/6e7ef756148c61580c9bf74cc16e30cf170e9c0d/tardis/src/db/reldb_client.rs#L184-L249

https://github.com/ideal-world/tardis/blob/1865b458d25c4c5f418630cd0e4c61e5c4d450b5/tardis/tests/test_reldb_client.rs#L520-L536
